pandas/core/groupby/groupby.py

```diff
@@ -1434,16 +1434,18 @@ def std(self, ddof: int = 1):
         Series or DataFrame
             Standard deviation of values within each group.
         """
-        return self._get_cythonized_result(
-            "group_var_float64",
-            aggregate=True,
-            needs_counts=True,
-            needs_values=True,
-            needs_2d=True,
-            cython_dtype=np.dtype(np.float64),
-            post_processing=lambda vals, inference: np.sqrt(vals),
-            ddof=ddof,
-        )
+        result = self.var(ddof=ddof)
```

Comment:
hmm you are reverting.

instead I would like to see if the generic function can be patched (e.g. something is fundamentally different).

cc @rhshadrach

Reply:
I don't see a way. `_get_cythonized_result` operates column-by-column, creating overhead. On data this size, we spend 25% in `grouper.group_info` and 40% in `_wrap_aggregated_output`. Within each of these operations, I don't see any easy wins. Only 27% of the time is spent doing the actual computation. 

The only way I see to improve performance is to change `_get_cythonized_result` to a 2d operation, operating on all columns at once. In the PR that caused this regression, that's the first thing I tried and found that it got too hairy. In the process, I realized that perhaps a better solution would be to incorporate the features of `_get_cythonized_result` (namely, pre- and post-processing) into `_cython_agg_general`.

Reply:
> hmm you are reverting.

yep. should have mentioned this in the OP.



> instead I would like to see if the generic function can be patched (e.g. something is fundamentally different).

OK. will close this for now. can reopen if a better solution is not forthcoming before the release.

Reply:
> OK. will close this for now. can reopen if a better solution is not forthcoming before the release.

or maybe better would be to backport this after 1.1rc if not fixed before.